caribou/data_collector/components/data_exporter.py

```diff
@@ -46,9 +52,9 @@ def _export_data(self, table_name: str, data: dict[str, Any], update_modified_re
             data_json: str = json.dumps(value)
             is_key_in_table: bool = self._client.get_key_present_in_table(table_name, key)
             if is_key_in_table:
-                self._client.update_value_in_table(table_name, key, data_json)
+                self._client.update_value_in_table(table_name, key, data_json, convert_to_bytes=convert_to_bytes)
```

Comment:
Out of curiosity, what would be the reason not to convert by default?

Reply:
Simply due to the way I have implemented update_value_in_table, with that approach since I don't automatically convert, there is a need to only convert specific tables that stores its content in byte format. But auto conversion will be a better approach and will be implemented in a later PR #341.